### ISSUE TITLE
Test set ground truth labels

### ISSUE THREAD

**Issue by rajeev595** [mcordts/cityscapesScripts#62]
Hi, Thanks for providing the scripts.

For evaluation on the test set, how are we supposed to extract the ground truth labels. Going through the test data, I have found only 3 classes ['out of roi', 'rectification border', 'ego vehicle'] none of which is in common with the training set labels. Please clarify.

Thanks.

**Comment by mcordts** [comment]
The test set is on purpose kept private. If you want to evaluate on the test set, you need to submit to our evaluation server (https://www.cityscapes-dataset.com/submit/). The public annotations only contain the data that is typically known without manual labeling, i.e. the ego vehicle and the evaluation ROI.

**Comment by serdarch** [comment]
> The test set is on purpose kept private. If you want to evaluate on the test set, you need to submit to our evaluation server (https://www.cityscapes-dataset.com/submit/). The public annotations only contain the data that is typically known without manual labeling, i.e. the ego vehicle and the evaluation ROI.

Dear Marius, 

How can I have the colored .png files with fine annotations.  I am using different techniques (such as label relaxation), so the submission of the prediction results to the system would not make any sense in my case.
I have to evaluate the network other than the format of the submission system. Can you please help?

Thanks in advance,
